apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChartLegends.vue

```diff
@@ -185,54 +127,27 @@ watch(() => state.groupByMenuItems, (after) => {
                 />
             </div>
             <template #no-data>
-                {{ t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.NO_ITEMS') }}
+                {{ $t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.NO_ITEMS') }}
             </template>
         </p-data-loader>
-        <cost-explorer-set-filter-modal v-model:visible="state.filterModalVisible"
-                                        :prev-selected-filters="costAnalysisPageState.filters"
-                                        :categories="CATEGORIES"
-                                        @confirm="handleUpdateFilters"
-        />
+        <p-button style-type="transparent"
+                  size="sm"
+                  font-weight="normal"
+                  @click="handleToggleAllLegends"
+        >
+            {{ state.showHideAll ? $t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.HIDE_ALL') : $t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.SHOW_ALL') }}
+        </p-button>
```

Comment:
nitpick: same here, I'll fix it.

```suggestion
            {{ state.showHideAll ? t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.HIDE_ALL') : t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.SHOW_ALL') }}
        </p-button>
```